applications/serial_lte_modem/doc/FTP_AT_commands.rst

```diff
@@ -130,29 +126,34 @@ Syntax
 
 ::
 
-   AT#XTFTP=<op>,<url>,<port>,<file_path>[,<mode>]
+   AT#XTFTP=<op>,<url>,<port>,<file_path>[,<mode>,<data>]
 
 * The ``<op>`` parameter can accept one of the following values:
 
   * ``1`` - TFTP read request using IP protocol family version 4.
-  * *Currently not supported* ``2`` - TFTP write request using IP protocol family version 4.
+  * ``2`` - TFTP write request using IP protocol family version 4.
   * ``3`` - TFTP read request using IP protocol family version 6.
-  * *Currently not supported* ``4`` - TFTP write request using IP protocol family version 6.
+  * ``4`` - TFTP write request using IP protocol family version 6.
 
 * The ``<url>`` parameter is a string.
   It indicates the hostname or the IP address of the TFTP server.
   Its maximum size is 128 bytes.
   When the parameter is an IP address, it supports both IPv4 and IPv6.
 * The ``<port>`` parameter is an unsigned 16-bit integer (0 - 65535).
-  It represents the TFTP service port on the remote server.
-  Default port 69 is applied if this parameter is omitted or set to ``0``.
+  It represents the TFTP service port on the remote server, which is 69 normally.
```

Comment:
This change makes it sound like there is no default value, and the user must instead apply the value that is normally 69. Is that the case?

Reply:
Yes no default value now